### PR TITLE
RD-5965 Bump `axios` and `follow-redirects` packages in backend to make Stage DB migration more robust

### PR DESCRIPTION
**Pull request by qooban**
## Description

A proper fix for app-crashing issue (https://github.com/follow-redirects/follow-redirects/issues/211 and RD-5663) was implemented in `follow-redirects` package and new version was released. Bumped `axios` package to get the latest `follow-redirects`.

Noticed that axios v1 is coming, so created RD-5969.

## Screenshots / Videos
N/A

## Checklist
- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3569/) - failed due to known issues (RD-5918, RD-5921, RD-5922), so I'm considering it as PASSED

## Documentation
N/A

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -97,6 +97,9 @@
         "sequelize": "^6.17.0",
         "sequelize-cli": "^6.4.1"
     },
+    "overrides": {
+        "follow-redirects": "^1.15.2"
```
Comment (kubama) [inline backend/package.json:101]: Would be good to somehow document the reason for this override - once it is no longer needed it should be removed.

**Review by kubama — APPROVED** [review]

**Comment by kubama** [comment]
I had a deeper look... please see https://github.com/cloudify-cosmo/cloudify-stage/pull/2178#issuecomment-1249016141 as I suspect it may also apply here. I think `override` may not be needed.

File: backend/package.json (hunk 1 above)
Comment (qooban) [reply backend/package.json]: Adding comments in `package.json` is problematic, but as I mentioned in the description:

> ... I have created [RD-5969](https://cloudifysource.atlassian.net/browse/RD-5969) and added a note to remove overrides once we move to axios v1.

**Comment by qooban** [comment]
> I had a deeper look... please see [#2178 (comment)](https://github.com/cloudify-cosmo/cloudify-stage/pull/2178#issuecomment-1249016141) as I suspect it may also apply here. I think `override` may not be needed.

Hmm... That was my original approach, but unfortunately `npm` did not pick up the latest version of `follow-redirects`, so I had to add `override`, but... I removed `override` right now and seems like it just works 🙂

**Review by kubama — APPROVED** [review]

File: backend/package.json (hunk 1 above)
Comment (qooban) [reply backend/package.json]: After recent changes (https://github.com/cloudify-cosmo/cloudify-stage/pull/2177/commits/f7d7b83e2d2cd4712cadf6067c557d392c17b889) removed that note from [RD-5969](https://cloudifysource.atlassian.net/browse/RD-5969).

**Pull request merged**